.eslintrc.json

```diff
@@ -42,9 +45,9 @@
       "error",
       {
         "require": {
-          "FunctionDeclaration": true,
-          "MethodDefinition": true,
-          "ClassDeclaration": true
+          "FunctionDeclaration": false,
+          "MethodDefinition": false,
+          "ClassDeclaration": false
         }
       }
     ]
```

Comment:
why are you disabling this rule

Reply:
We did it because we thought jsdocs were unnecessary but I guess we'll have to undo that.